Source/Streamstone.Tests/Scenarios/Tracking_entity_changes.cs

```diff
@@ -577,15 +586,21 @@ yield return
             }
         }
 
-        void InsertTestEntity(TestEntity entity)
+        void InsertTestEntity(ITableEntity entity)
```

Comment:
I could have renamed this to `InsertEntity`, but the diff would be much less meaningful.